backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/events/DocumentWebsocketPublisher.java

```diff
@@ -163,6 +163,7 @@ private static final void collectFrom(final JSONDocumentChangedWebSocketEventCol
 		}
 
 		final DocumentId documentId = event.getId();
+		collector.staleRootDocument(windowId, documentId);
```

Comment:
@teosarca  suggested this fix, to update the UI even when there's an inconsistency (for ex when selecting bpartner with missing price system)